presto-docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -158,6 +158,9 @@ For example, the mapping list might have URL prefix ``s3://abc/xyz/`` followed b
 ``s3://abc/`` to allow different configuration for a specific path within a bucket
 than for other paths within the bucket. You can set default configuration by not
 including any match criteria for the last entry in the list.
+The default configuration could contain only one property
```

Comment:
This is not true, default configuration could contain a dedicated role or credentials or it could contain `useClusterDefault`.

Please all mention when is going to happen where is no default configuration.

Reply:
changed